### ISSUE TITLE
[StoreSample] An error Occured

### ISSUE THREAD

**Issue by RealStyle12** [Microsoft/DesktopBridgeToUWP-Samples#23]
I get the: "update AppxManifest.xml". I published the app to windows store, so app identity can't be wrong. ( the manifest is checked by that process ). Also I have 2 add-ons "Consumable (Store-managed)" in this app.
Also I added using Windows.ApplicationModel and checked if all the data from the manifest is correct.

PS:  after "StoreContext storeContext = StoreContext.GetDefault();" storeContext is still null.

**Comment by RealStyle12** [comment]
Any updates ?

**Comment by vladimp** [comment]
Hi, 

Sorry for the delay, can you please share the full error you are receiving, Is it a build or a run time error?
If you can please share a sample of your code and your manifest - to vladimp@microsoft.com

Thanks,
Vlad

**Comment by RealStyle12** [comment]
Hello,

It is a runtime error.
Thank you for the update, I have just packed and uploaded the source.

Thanks,
Stefan

**Comment by vladimp** [comment]
Hi!

Did you upload your app to the store?
You can only test these APIs after uploading to the Store. The same is true for regular UWP apps as well.

So the recommended workflow would be:
1.	Request permission for ‘runFullTrust’
2.	Publish to the store as hidden/private beta
3.	Do your testing
4.	Unhide the app in the store once all testing is done

Alternatively, If you don’t want to request the runFullTrust permission just yet, you can publish a regular UWP app on the Store as hidden and then leverage the same identity on a converted one that is running on your PC. What’s important is that the identity of the app on the Store matches the one you’re testing.

Thanks!
Vlad

**Comment by RealStyle12** [comment]
Hello,

Thank you for the information, I already done that.
Apparently the problem was from my computer configuration. I gave the project solution based on this repo to a work colleague and apparently at him worked perfectly. 

All the best,
Stefan

**Comment by vladimp** [comment]
Great! Glad it worked for you.
